advanced_survey/views.py

```diff
@@ -62,8 +92,8 @@ def save_survey(request): # pylint: disable=R0912,R0915
         pages = json.loads(request.body)
         page_counter = 0
         for page in pages:
+            id_lookup = dict()
             for question in page:
-
                 db_question = Question()
                 if str(question['id']).isnumeric():
```

Comment:
here we load question from db if the id is numerc, after your changes in the frontend, all ids become integer, so please edit this code

Reply:
That change was rolled back